projects/igniteui-angular/src/lib/grids/summaries/summary-cell.component.html

```diff
@@ -12,7 +12,11 @@
 
                 <span class="igx-grid-summary__label" [title]="summary.label">{{ translateSummary(summary) }}</span>
                 <span class="igx-grid-summary__result" [title]="summary.summaryResult">
-                    {{ summary.summaryResult }}
+                    {{
+                        (isDateSummary(summary.key) && column.formatter)
```

Comment:
Does `isDateSummary` covers the case of a custom summary key? If not, the column dataType could be considered (assuming we wont be covering another case - when summary data type differs from column data type).